package.json

```diff
@@ -25,6 +25,7 @@
     "typescript": "~2.4.2"
   },
   "scripts": {
+    "doc": "esdoc",
```

Comment:
Not sure what happened to my original feedback. I was saying thanks for adding this script and was thinking it'd be good to add `yarn lint` and `yarn doc` to the contributing docs under the [style guidelines section](https://github.com/sendgrid/sendgrid-nodejs/blob/master/CONTRIBUTING.md#style-guidelines--naming-conventions).

Reply:
Thanks for the feedback. I have made the suggested change.